semantic-python/semantic-python.cabal

```diff
@@ -18,17 +18,33 @@ extra-source-files:  README.md
 
 tested-with:         GHC == 8.6.4
 
-library
-  exposed-modules:
-    Language.Python.Core
-  -- other-modules:
-  -- other-extensions:
-  build-depends:       base ^>=4.12.0.0
+common haskell
+  default-language:    Haskell2010
+  default-extensions:  DataKinds
+                     , DeriveFoldable
+                     , DeriveFunctor
+                     , DeriveGeneric
+                     , DeriveTraversable
+                     , FlexibleContexts
+                     , FlexibleInstances
+                     , MonadFailDesugaring
+                     , MultiParamTypeClasses
+                     , OverloadedStrings
+                     , RecordWildCards
+                     , StandaloneDeriving
+                     , StrictData
+                     , TypeApplications
```

Comment:
I’d prefer not to use `default-extensions` if at all possible, since it makes parsing harder for both programs and me.